src/content/ytui-music.mdx

```diff
@@ -0,0 +1,5 @@
+title: Ytui-music
+excerpt: Listen to youtube from terminal ui
+tags: rust, tui, terminal, youtube, music
+featuredImg: '../images/sudipghimire533/image.png'
+projectLink: https://github.com/sudipghimire533/ytui-music
```

Comment:
```suggestion
---
title: YouTube UI Music
excerpt: Listen to youtube from terminal ui
tags: rust, tui, terminal, youtube, music
featuredImg: '../images/sudipghimire533/image.png'
projectLink: https://github.com/sudipghimire533/ytui-music
---
```

I have a suggestion for you, you need to use `---` to wrap the content for frontmatter usecase. You can use the suggested code block and update the PR. It will be awesome if you could also add other information on body section. For your reference please check `nilo.mdx` file.

Also, you have mentioned, `featuredImg` which you haven't pushed. You need to add your image under `image` directory by creating another folder with your username.